### ISSUE TITLE
Deno.readAll returning empty Uint8Array possible bug?

### ISSUE THREAD

**Issue by gabrielgrover** [denoland/deno#6241]
I am not sure if I am just using this function incorrectly or if this is a bug.  

```
  const body = JSON.stringify({ hello: "world" });
  const req = new Request("hello.com", { method: "POST", body });
  Deno.readAll(req.body.getReader()).then(console.log);

  -> Uint8Array(0) []
```
Is the reader from the call req.body.getReader() not compatible with Deno.readAll?  Not sure why this array is empty when there is clearly data in the req.body readable stream.

**Comment by SyrupThinker** [comment]
The streams are incompatible. This not throwing an exception is a bug that was solved in #6161, which should be part of the next release.

**Comment by gabrielgrover** [comment]
Thanks @SyrupThinker!  Closing this out.
